src/components/Editor/EditorSlice.js

```diff
@@ -131,11 +139,11 @@ export const EditorSlice = createSlice({
         state.project.image_list = []
       }
       state.loading='success'
-      if (state.openFiles.length === 0) {
+      if (state.openFiles[0].length === 0) {
```

Comment:
again I might be missing something...but even though it's an index, I'd be inclined to assign this to a variable so it's clearer form the code what's going on? something like `editorPanel` or `primaryPanel`, still trying to get my head around this and seeing a hard coded index is a bit confusing

Reply:
Yeah so it's meant to be checking whether any files are open and if not opening `main.py`/`index.html`, but was just checking the first panel, but I've just changed it to `openFiles.flat()` rather than `openFiles[0]` as that seems to make more sense for what we were trying to do there